conventional_precommit_linter/hook.py

```diff
@@ -222,6 +231,7 @@ def parse_args(argv: List[str]) -> argparse.Namespace:
     parser.add_argument('--subject-max-length', type=int, default=72, help="Maximum length of the 'Summary'")
     parser.add_argument('--body-max-line-length', type=int, default=100, help="Maximum length of the 'Body' line")
     parser.add_argument('--summary-uppercase', action='store_true', help="'Summary' must start with an uppercase letter")
+    parser.add_argument('--allow-uppercase-in-optional-scope', action='store_true', help='Allow uppercase letters in the optional scope.')
```

Comment:
This is too long .... can you please change it to something more reasonable (e.g. `--scope-case-insensitive`)